components/scream/src/physics/shoc/atmosphere_macrophysics.cpp

```diff
@@ -141,7 +146,9 @@ void SHOCMacrophysics::initialize_impl (const util::TimeStamp& t0)
   const int nlevi_packs = ekat::npack<Spack>(m_num_levs+1);
   const int num_tracer_packs = ekat::npack<Spack>(m_num_tracers);
 
-  view_1d wpthlp_sfc("wpthlp_sfc",m_num_cols),
+  view_1d host_dx("host_dx",m_num_cols),
+          host_dy("host_dy",m_num_cols),
+          wpthlp_sfc("wpthlp_sfc",m_num_cols),
```

Comment:
What do `wpthlp_sfc` and the other `_sfc` names stand for? Any chance we can use a few more chars and make it clearer?

Reply:
I know this stuff was here before this PR, so it's not a deal breaker. But since it would be a simple find+replace op, if there's the will and opportunity to make it more readable, we might as well take it.

Reply:
_sfc stands for surface. I personally don't have a problem with that. I agree that wpthlp is a mouthful. It stands for $\overline{w'\theta_l'}$, though, which is hard to write in English at all. Maybe vert_liq_pot_T_flux? I think these names were created by @bogensch so maybe he has an opinion on this.

Reply:
From what I understand these are surface values. I could replace with `_surface`.

Reply:
wpthlp was intended to stand for w_prime_thl_prime, but since it's not clear to others outside of turbulence I have no objections to something like vert_liq_pot_T_flux